### DISCUSSION TITLE
bug: StatusBar unit test 'not implemented on web'

### DISCUSSION THREAD

**Discussion by ludonoel1** [ionic-team/capacitor#6488] [Q&A]
### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 4.7.3
  @capacitor/core: 4.7.3
  @capacitor/android: 4.7.3
  @capacitor/ios: 4.7.3

Installed Dependencies:

  @capacitor/cli: 4.1.0
  @capacitor/core: 4.1.0
  @capacitor/android: 4.1.0
  @capacitor/ios: 4.1.0

```

### Platform(s)

web

### Current Behavior

I'm trying to unit test my code with Karma Jasmine but whatever I'm using (spyon / spyonObject / returnValue etc..) I get this error : "Statusbar plugins is not implemented on web"
I have tried mock from __mock__/@capacitor but I get other issues
./__mocks__/@capacitor/status-bar:4:18 - Error: Module parse failed: Unexpected token (4:18)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| export const StatusBar = {
>   setStyle(options: StyleOptions): void {},
|   setBackgroundColor(options: BackgroundColorOptions): void {}
| 
### Expected Behavior

 My test passes

### Other Technical Details

`npm --version` output:

`node --version` output:

`pod --version` output (iOS issues only):

**Comment by jcesarmobile** [comment]
Converted to discussion, this is not a Capacitor issue but a problem with your mocks/configuration.
Capacitor doesn't provide mocks.

I can't help you with that, I guess you already checked https://capacitorjs.com/docs/guides/mocking-plugins?

**Reply by ludonoel1** [reply]
Yes. 
Is it the only solution to mock StatusBar ?

**Reply by ludonoel1** [reply]
`./src/shared/utils/function.utils.ts:367:17-28 - Error: export 'Style' (imported as 'Style') was not found in '@capacitor/status-bar' (possible exports: StatusBar)

./src/shared/utils/function.utils.ts:381:17-28 - Error: export 'Style' (imported as 'Style') was not found in '@capacitor/status-bar' (possible exports: StatusBar)

./src/shared/utils/function.utils.ts:395:17-27 - Error: export 'Style' (imported as 'Style') was not found in '@capacitor/status-bar' (possible exports: StatusBar)

Error: src/shared/utils/function.utils.ts:2:21 - error TS2305: Module '"@capacitor/status-bar"' has no exported member 'Style'.

2 import { StatusBar, Style } from "@capacitor/status-bar";
                      ~~~~~
`

I get these errors
